content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -1234,11 +1235,11 @@ contributors. Follow the instructions in the SIG-Windows
 
 * `kubectl port-forward` fails with "unable to do port forwarding: wincat not found"
 
-  This was implemented in Kubernetes 1.15 by including wincat.exe in the
-  pause infrastructure container `mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
+  Port forwarding support for Windows was added in Kubernetes 1.15 by including wincat.exe in the
+  [pause infrastructure container](#Pause-Image).
```

Comment:
We can omit this; Kubernetes versions before 1.15 are out of support.

Reply:
Updated/removed 1.15 reference.